test/e2e_node/mirror_pod_test.go

```diff
@@ -160,7 +161,7 @@ func checkMirrorPodDisappear(cl *client.Client, name, namespace string) error {
 	if errors.IsNotFound(err) {
 		return nil
 	}
-	return err
+	return goerrors.New("Pod Not Disappear")
```

Comment:
1. Why not use `errors` directly?
2. We should not user capital in error string: https://github.com/golang/go/wiki/CodeReviewComments#error-strings